code/modules/client/preference/preferences.dm

```diff
@@ -551,6 +552,9 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 		if(jobban_isbanned(user, rank))
 			HTML += "<del>[rank]</del></td><td><b> \[BANNED]</b></td></tr>"
 			continue
+		if(!has_exp_for_job(user, job.title))
+			HTML += "<del>[rank]</del></td><td> \[TIME]</td></tr>"
```

Comment:
This should probably show the time and department. Like, [1h as Crew], [3h as Science], [10h as Command], etc.